examples/next-js-on-demand-isr/README.md

```diff
@@ -0,0 +1,158 @@
+# Next.js + Builder.io On-Demand ISR
+Example of generating pages with Builder using nextjs ISR with [on-demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation)
+
+## Prerequisites
+
+Before using this example, make sure you have the following:
+
+- A [Builder.io](builder.io) account. Check out the [plans](https://www.builder.io/m/pricing), which range from our free tier to custom.
+- [npm](https://www.npmjs.com/)
+
+A basic understanding of the following is helpful too:
+
+- The [command line](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line)
+- JavaScript and [React](https://reactjs.org/)
+- [Next.js](https://nextjs.org/)
+- What it means to [clone a GitHub repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+## Overview
+
+To use this project, you need to do three things:
+
+1. Get a copy of this repo.
+1. Create a corresponding space in your account on [Builder.io](builder.io).
+1. Connect the two.
+
+The next sections walk you through each step.
+
+### Clone this project
+
+This example provides you with a ready-made application that you can copy locally and configure for use with your Builder.io account.
+
+1. At the command line, run the following command to create a local copy of the Builder.io repo:
+
+   ```shell
+   git clone https://github.com/BuilderIO/builder.git
+   ```
+
+1. Change into the example application by using the `cd` command:
+
+   ```shell
+   cd builder/examples/next-js-on-demand-isr
+   ```
+
+1. Open this directory in your favorite code editor.
+   You'll come back here when you're ready to connect your application to your Builder space.
+
+### Generating your Builder.io space
+
+If you've just created your Builder.io account and logged in for the first time, Builder prompts you to create a space with a new Builder site or add Builder to an existing application.
```

Comment:
Not sure this is relevant anymore since we deprecated hosting.

Reply:
true, this whole section feels extraneous, i'l remove